contracts/contracts/solidity/StarknetMessaging.sol

```diff
@@ -0,0 +1,85 @@
+// SPDX-License-Identifier: Apache-2.0.
```

Comment:
This contract looks to be a slimmer version from origin: https://github.com/starkware-libs/starkgate-contracts/blob/main/src/starkware/starknet/solidity/StarknetMessaging.sol

Did we make any changes to the code? If we didn't, then let's import and use it from origin instead.

Reply:
I modified it by adding IStarknetCore so I been able to delete MockStarknetValidator.sol